src/client/refresh/soft/sw_main.c

```diff
@@ -1469,6 +1470,9 @@ static int RE_PrepareForWindow(void)
 	return flags;
 }
 
+// Declared in vid/header/ref.h
+refexport_t	re;
```

Comment:
This is not supposed to be a global variable in the ref DLLs.
It *is* a global variable in the executable (see vid.c) (and is set from the return value of GetRefAPI() from the ref DLL)